graphula-core/src/Graphula.hs

```diff
@@ -101,6 +104,21 @@ type MonadGraphula m =
   , MonadThrow m
   )
 
+-- | A constraint over lists of nodes for 'MonadGraphula', and 'GraphulaNode'.
+--   Helpful for defining utility functions over many nodes.
```

Comment:
FWIW: I'd format this as:

```hs
-- | A constraint over lists of nodes for @'MonadGraphula'@ and @'GraphulaNode'@
--
-- Helpful for defining utility functions over many nodes.
--
-- @
-- ...
-- @
--
type family ...
```

Which follow these guides:

- Have a short first line, not punctuated (like a commit message)
- Keep a line of whitespace before the "body" (like a commit message), which makes it not-annoying to
- Align the "body" normally
- For docs with bodies, put another line of "whitespace" before the function (I think it's also generally advised to surround code blocks with "whitespace", which applies here as well)
- Use proper "markdown" (the haddock kind)